rules/integrations/azure/initial_access_azure_active_directory_high_risk_signin_atrisk_or_confirmed.toml

```
@@ -15,13 +15,61 @@ index = ["filebeat-*", "logs-azure*"]
+
+Microsoft Identity Protection is an Azure AD security tool that detects various types of identity risks and attacks.
+
+This rule will look for the occurrence of events produced by the Microsoft Identity Protection with a risk state equal
+to `confirmedCompromised` or `atRisk`.
+
+#### Possible investigation steps
+
+- Identify the Risk Detection that triggered the event. A list with descriptions can be found [here](https://docs.microsoft.com/en-us/azure/active-directory/identity-protection/concept-identity-protection-risks#risk-types-and-detection).
+- Identify the user account involves and validate whether the suspicious activity is normal for the given user. 
+  - Consider the source IP address and geolocation for the involved user account. Do they look normal?
+  - The device used to sign-in, is it registered or compliant?
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Contact the account owner and confirm whether they are aware of this activity.
+- Check if this operation is done under change management and approved according to the organization's policy.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking access to servers,
+services, and data accessed by the account in the last 24 hours.
+
+### False positive analysis
+
+- If this activity is expected and noisy in your environment, consider adding exceptions — preferably with a combination
+of user and device conditions.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Identify the possible impact of the incident and prioritize accordingly; the following actions can help you gain context:
+    - Identify the account role in the cloud environment.
+    - Identify the services or servers involved criticality.
+    - Work with your IT team to identify the impact on users.
+    - Identify if the attacker is moving laterally and compromising other accounts, servers, or services.
+    - Identify if there are any regulatory or legal ramifications related to this activity.
+- Disable or limit the account during the investigation and response.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are
+identified. Reset passwords or delete API keys as needed to revoke the attacker's access to the environment. Work with
+your IT teams to minimize the impact on the business operation during these actions.
+- Check if unauthorized new users were created, remove new unauthorized accounts, and request password resets for other
+IAM users.
+- Consider enabling multi-factor authentication for users.
+- Follow security best practices [outlined](https://docs.microsoft.com/en-us/azure/security/fundamentals/identity-management-best-practices) by Microsoft.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
```

```suggestion
- Identify the initial vector abused by the attacker and take action to prevent reinfection via the same vector.
```
Applies elsewhere in the PR